pom.xml

```
@@ -122,11 +122,8 @@
                     <failOnWarning>false</failOnWarning>
                     <compilerArgs>
                         <arg>-XDcompilePolicy=simple</arg>
-                        <arg>
-                            -Xplugin:ErrorProne
-                            <!-- TODO: Enable DoubleBraceInitialization flag -->
-                            -Xep:DoubleBraceInitialization:OFF
-                        </arg>
+                        <arg>-Xplugin:ErrorProne
+                             -Xep:DoubleBraceInitialization:ERROR</arg>
```

The default severity of `DoubleBraceInitialization` is ERROR. https://errorprone.info/bugpattern/DoubleBraceInitialization

```suggestion
                        <arg>-Xplugin:ErrorProne</arg>
```